app/controllers/ui_job_wizard_controller.rb

```diff
@@ -57,4 +57,19 @@ def resources
     render :json => { :results =>
       resource_list.sort_by { |r| r[:name] }.take(100), :subtotal => resource_list.count}
   end
+
+  def job_invocation
+    job = JobInvocation.authorized.find(params[:id])
+    composer = JobInvocationComposer.from_job_invocation(job, params).params
+    job_template_inputs = JobTemplate.authorized.find(composer[:template_invocations][0][:template_id]).template_inputs_with_foreign
+    inputs = Hash[job_template_inputs.map { |input| ["inputs[#{input[:name]}]", (composer[:template_invocations][0][:input_values].select { |value| value[:template_input_id]== input[:id] })[0][:value]] } ]
```

Comment:
Please do use `find` over `select {...}[0]`, there's no need to collect all the elements matching a predicate if we only need the first one